Tests/CylindricalExtractionTest/SimulationParameters.hpp

```diff
@@ -0,0 +1,58 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef SIMULATIONPARAMETERS_HPP_
+#define SIMULATIONPARAMETERS_HPP_
+
+// General includes
+#include "ChomboParameters.hpp"
+#include "GRParmParse.hpp"
+
+// Problem specific includes
+#include "CylindricalExtraction.hpp"
+
+class SimulationParameters : public ChomboParameters
+{
+  public:
+    SimulationParameters(GRParmParse &pp) : ChomboParameters(pp)
+    {
+        read_params(pp);
+    }
+
+  private:
+    void read_params(GRParmParse &pp)
+    {
+        // extraction params
+        dx.fill(coarsest_dx);
+        origin.fill(coarsest_dx / 2.0);
+
+        // Extraction params
+        pp.load("num_extraction_radii", extraction_params.num_extraction_radii,
+                1);
+
+        if (pp.contains("extraction_radii"))
+        {
+            pp.load("extraction_radii", extraction_params.extraction_radii,
+                    extraction_params.num_extraction_radii);
+        }
+        else
+        {
+            pp.load("extraction_radius", extraction_params.extraction_radii, 1,
+                    L / 4);
+        }
+	pp.load("num_points_z", extraction_params.num_points_z, 33);
+	pp.load("z_length", extraction_params.z_length, L);
+        pp.load("num_points_phi", extraction_params.num_points_phi, 25);
+        pp.load("extraction_center", extraction_params.center,
+                {0.5 * L, 0.5 * L, 0.5 * L});
```

Comment:
Same. {0.5*L} to 'center'

Reply:
I have removed the CylindricalExtractionTest as I think it's a bit redundant (see my comment above) but I have fixed this in the `SimulationParameters` for the SphericalExtractionTest